.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# How to contribute to MFTF docs
+
+MFTF documentation is kept within the /docs/ folder in this repository.
+We welcome contributions to the documentation.
+This page describes the process for submitting documentation and serves as a template for a properly written content.
+
+The contribution workflow for the Magento functional testing framework (MFTF) documentation is the same as submitting code.
+
+1. Create a branch from the `develop` branch in the [MFTF repo][].
```

Comment:
Change "repo" to "repository". Avoid abbreviations.

Reply:
I am slowly learning.